src/helpers/index.tsx

```diff
@@ -20,18 +22,35 @@ export async function getMarketPrice({ networkID, provider }: { networkID: numbe
   return marketPrice;
 }
 
+export async function getTokenPrice() {
+  let tokenId = "olympus";
+  const resp = await axios.get(`https://api.coingecko.com/api/v3/simple/price?ids=${tokenId}&vs_currencies=usd`);
+  let tokenPrice: number = resp.data[tokenId].usd;
+  return tokenPrice;
+}
+
 export function shorten(str: string) {
   if (str.length < 10) return str;
   return `${str.slice(0, 6)}...${str.slice(str.length - 4)}`;
 }
 
+export function formatCurrency(c: number, precision: number | undefined) {
```

Comment:
I like using default parameters if we're expecting an "undefined" so it would be
`formatCurrency(c: number, precision = 0)`

Reply:
aaah ok.